### ISSUE TITLE
Add a status command

### ISSUE THREAD

**Issue by woctezuma** [Citrinate/FreePackages#8]
It could be useful to have a status command to print the following information:

```
2023-10-07 10:12:08|dotnet-6752|INFO|Bot|ProcessQueue() Pausing free package activations until 11:13:03
```

And maybe, if the activations are not paused, then the current state, e.g. 25/N packages activated in the past hour, where N is the user-input rate-limit.

**Comment by Citrinate** [comment]
Added 3 commands.  New version should be out next week.  You can test it here: 
[FreePackages V1.1.0.zip](https://github.com/Citrinate/FreePackages/files/12838804/FreePackages.zip)

Command | Access | Description
--- | --- | ---
`clearfreepackagesqueue [Bots]`|`Master`|Removes everything from the given bot's free packages queue
`queuestatus [Bots]`|`Master`|Prints the status of the given bot's free packages queue
`queuelicense [Bots] <Licenses>`|`Master`|Adds given `licenses`, explained [here](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/Commands#addlicense-licenses), to the given bot's free packages queue
`queuelicense^ [Bots] <Licenses>`|`Master`|Adds given `licenses`, explained [here](https://github.com/JustArchiNET/ArchiSteamFarm/wiki/Commands#addlicense-licenses), to the given bot's free packages queue using that bot's package filters

#### Command Aliases

Command | Alias |
--- | --- |
`queuestatus asf`|`qsa`
`queuelicense`|`queuelicence`, `qlicense`, `qlicence`
`queuelicense^`|`queuelicence^`, `qlicense^`, `qlicence^`

**Comment by woctezuma** [comment]
Perfect! Thank you!

**Comment by woctezuma** [comment]
There is one issue: if one of the bots does not use the plugin, then the command `qsa` fails.

**Comment by woctezuma** [comment]
Adding `  "EnableFreePackages": false,` to the configuration of `Bot_NO_PLUGIN` does not solve the issue.

**Comment by Citrinate** [comment]
This should fix that: [FreePackages V1.1.0.zip](https://github.com/Citrinate/FreePackages/files/12838925/FreePackages.zip)

**Comment by woctezuma** [comment]
Thanks! This works!
